config/configuration.json

```diff
@@ -799,6 +799,7 @@
   "SinkIslandCheck": {
     "tree.size": 50,
     "minimum.highway.type": "service",
+    "filter.pedestrian.network": true,
```

Comment:
Is the default config going to be true for "filter.pedestrian.network"?

Reply:
It should be false to maintain baseline outputs. Thanks